gensim/sklearn_api/d2vmodel.py

```diff
@@ -145,6 +145,11 @@ def __init__(self, dm_mean=None, dm=1, dbow_words=0, dm_concat=0, dm_tag_count=1
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
 
+    def _get_first(some_iterable):
```

Comment:
Methods should accept `self` as the first parameter.

Why is this a method? It can be a pure function, either at the module level or internal.

Reply:
done